docs/guides/user/docs/advanced/index.md

```diff
@@ -2,4 +2,15 @@
 
 This section describes facilities supposed to be used by advanced users.
 
-- [Role-Based Visibility](role-based-visibility.md)
\ No newline at end of file
+### Unprivileged Users
+
+If a user owns the global administration role (`ROLE_ADMIN`), he is a privileged users in means of having access
```

Comment:
*user

Reply:
You can avoid the "he" by writing "If users own the global administrator role, they are privileged users", by the way.